src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  margin-top: 0;
```

Comment:
checklist point 8 (but I am not sure if margins with 0 value are also forbidden for BEM blocks :) )